### ISSUE TITLE
Have eq_ but no neq_

### ISSUE THREAD

**Issue by Zitrax** [nose-devs/nose#882]
Any reason to not add a neq_ or similar as a shorthand for assert_not_equal ?

It feels a bit asymmetrical to have ```eq_(…)``` mixed with ```assert_not_equal(…)``` in the code.

**Comment by jszakmeister** [comment]
No, there's probably not any good reason for it--just that it was never provided.  Why does everything have to be symmetrical? ;-)  I'd consider a patch for it though.  `nose.tools.trivial` would be the place to look, and tests for it would be nice too.
